controllers/user.controller.js

```diff
@@ -26,6 +27,10 @@ exports.createUser = (req, res, next) => {
     email,
     role,
     accountStatus,
+    dateOfBirth,
+    numberOfLogins,
+    totalGameplayTime,
+    communityInteractionFrequency
```

Comment:
`numberOfLogins,
 totalGameplayTime,
 communityInteractionFrequency`
Are not required to create the user document here, and since we don't expect to receive anything when the user is created for these fields, we shouldn't be passing these null objects.